introduction.md

```diff
@@ -4,3 +4,4 @@ I'm :
 * Git
 * JavaScript
 * Ruby
+* HTML5
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?